salt/returners/etcd_return.py

```diff
@@ -113,6 +113,13 @@ def save_load(jid, load):
     )
 
 
+def save_minions(jid, minions):  # pylint: disable=unused-argument
+    '''
+    Included for API consistency
+    '''
+    pass
```

Comment:
The only reason that I did not is that we're not already saving the minion list for this returner.